install/terraform/modules/aks/aks.tf

```diff
@@ -12,46 +12,52 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-provider "azuread" {
-  version = "=0.4.0"
-}
-
-provider "azurerm" {
-  version = "=2.2.0"
 
-  features {}
-}
-
-provider "random" {
-  version = "~> 2.2"
-}
-
-# Create Service Principal password
-resource "azuread_service_principal_password" "aks" {
-  end_date             = "2299-12-30T23:00:00Z" # Forever
-  service_principal_id = azuread_service_principal.aks.id
-  value                = random_string.password.result
-}
-
-# Create Azure AD Application for Service Principal
-resource "azuread_application" "aks" {
-  name = "agones-sp"
-}
-
-# Create Service Principal
-resource "azuread_service_principal" "aks" {
-  application_id = azuread_application.aks.application_id
+terraform {
+  required_providers {
+    # azuread = {
+    #   source = "hashicorp/azuread"
+    #   version = "~> 1.5"
+    # }
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.63"
+    }
+    # random = {
+    #   source  = "hashicorp/random"
+    #   version = "~> 3.1"
+    # }
+  }
+  required_version = ">= 0.12.26"
 }
 
-# Generate random string to be used for Service Principal Password
-resource "random_string" "password" {
-  length  = 32
-  special = true
-}
+# 2021.06.16-WeetA34: Commented because not used for now
```

Comment:
Can you remove this comment or tell us what the commented out config is useful for?

Reply:
I see that this is the previous version which I added before. It might be obsolete and not needed so I think you can simply remove old version.